base/commandLine.cpp

```diff
@@ -334,7 +334,7 @@ void registerDefaults() {
 	ConfMan.registerDefault("enable_unsupported_game_warning", true);
 	ConfMan.registerDefault("enable_unsupported_addon_warning", true);
 
-#ifdef USE_FLUIDSYNTH
+#if defined(USE_FLUIDSYNTH) || defined(USE_FLUIDLITE)
```

Comment:
Same here, this should go in the same commit as the Makefile.common change

Reply:
I split this into a separate commit